src/api/.rubocop.yml

```diff
@@ -69,6 +69,10 @@ Metrics/BlockLength:
     - 'spec/**/*'
     - '**/*.rake'
 
+Metrics/ClassLength:
+  Exclude:
+    - 'app/controllers/webui/package_controller.rb'
```

Comment:
I wonder why this is needed now. This controller has been huge for already a loooooong time. I would prefer to increase the value of `Max` here instead:

https://github.com/openSUSE/open-build-service/blob/41aa41acd7b3ab38b4661d6dd5bfcad6029d1c35/src/api/.rubocop_todo.yml#L189-L190

We don't create huge classes so this will be tackle in the future whenever we split up that controller.